src/storybook-components/_data/demoData.js

```diff
@@ -145,7 +145,8 @@ const demoData = {
     ]),
     moreLink: object('More Link', { text: 'See more', url: '#' }),
     theme: selectV2('Theme', {dark: 'dark', light: 'light'}, 'light'),
-    showLinks: boolean('Show Links', true)
+    showLinks: boolean('Show Links', true),
+    logo: text('Logo', 'https://placehold.it/130x100')
```

Comment:
Can you please make a static logo example in root `static/` folder? That will be faster.